api/Navigator.json

```diff
@@ -3332,6 +3332,54 @@
           }
         }
       },
+      "userAgentData": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/ua-client-hints/#dom-navigatorua-useragentdata",
+          "support": {
+            "chrome": {
+              "version_added": "90"
+            },
+            "chrome_android": {
+              "version_added": "90"
+            },
+            "edge": {
+              "version_added": "90"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "76"
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
```

Comment:
No WebView?

Reply:
Apparently the WebView version we have in our data is an older version (I haven't updated my testing Android in a while).  Lemme collect results for the latest WebView, one moment...

Reply:
There seems to be a bug that's preventing people from updating their WebView (or Chrome) to the latest release on Android devices, and it's affecting my device as well.  I will not be able to test in the latest WebView until that issue is resolved.  Should I just mark it as `null` for now?

Reply:
Via [source.chromium.org](https://source.chromium.org/chromium/chromium/src/+/main:android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt;l=153;drc=19e16336f47125b3e4b5c34cac0e93a2f4c921cf) I found my way to https://crbug.com/921655, turns out this isn't enabled in WebView. Let's link https://crbug.com/921655 though.